pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -11,6 +11,7 @@
 , useProfiles ? true
 , preferGtk2 ? false
 , settings32Bit ? false
+, ibtSupport ? null
```

Comment:
```suggestion
, ibtSupport ? lib.versionAtLeast version "530"

Reply:
`lib` isnt bound there, the secondary attrset is what gets called with `callPackage`

Reply:
Oh, then just default to false and keep the logic in the assignment below.